### ISSUE TITLE
Craft the Arkenplate quest.

### ISSUE THREAD

**Issue by robotliar** [kyber-6/fantasy-mc-fabric-modpack#80]
### Describe the bug

Craft the Arkenplate quest unable to be completed. Required min dependencies needed exceeded the available dependent quests.

### Reproduction Steps

Complete the two required quests before the Craft the Arkenplate quest. Craft the Arkenplate quest will be still greyed out and uncompletable.

### Mod pack version

v 2.9.3b

**Comment by KevzCz** [comment]
yes i've seen someone said this already, an update will fix it but for now you can join the discord server and follow this to fix it https://discord.com/channels/872788557553610802/872803129580736543/1240278258751242393

**Comment by etendards** [comment]
That link doesnt work for me.  Any idea on how to fix it?  I am on the CDU server.

> yes i've seen someone said this already, an update will fix it but for now you can join the discord server and follow this to fix it https://discord.com/channels/872788557553610802/872803129580736543/1240278258751242393

**Comment by etendards** [comment]
Is this mod pack dead?

**Comment by KevzCz** [comment]
I am currently busy with real life stuff, so updates have slowed down.

**Comment by etendards** [comment]
> I am currently busy with real life stuff, so updates have slowed down.

Totally  understand.  This is a great Mod Pack and I have been spending LOTS of time enjoying it.  Keep up the good work!
